gradle/plugins/src/main/kotlin/com.hedera.gradlebuild.jpms-modules.gradle.kts

```diff
@@ -50,86 +42,77 @@ dependencies.components {
             "org.codehaus.mojo:animal-sniffer-annotations"
         )
 
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-netty") {
-        params(grpcComponents + annotationLibraries)
-    }
-    withModule<AddDependenciesMetadataRule>("io.grpc:grpc-netty") { params(grpcModule) }
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-protobuf") {
-        params(grpcComponents + annotationLibraries)
-    }
-    withModule<AddDependenciesMetadataRule>("io.grpc:grpc-protobuf") { params(grpcModule) }
-    withModule<RemoveDependenciesMetadataRule>("io.grpc:grpc-protobuf-lite") {
-        params(
-            grpcComponents + annotationLibraries + listOf("com.google.protobuf:protobuf-javalite")
-        )
+    module("io.netty:netty-transport-native-epoll") {
+        addFeature("linux-x86_64")
+        addFeature("linux-aarch_64")
```

Comment:
What is `addFeature` doing?

Reply:
There is Javadoc on the `addFeature` that says:

> Make the Jar with the give 'classifier' known as _Feature Variant_ so that it can be selected via capability
in a dependency declaration.
See: https://docs.gradle.org/current/userguide/component_metadata_rules.html#making_different_flavors_of_a_library_available_through_capabilities
See: https://blog.gradle.org/optional-dependencies

In short: the information that the classified Jars exist is added to the metadata which then allows the _java-module-dependencies_ plugin to address the Jars as individual modules.